solver/src/settlement/settlement_encoder.rs

```diff
@@ -1,6 +1,6 @@
-use super::{Interaction, Trade};
+use super::{Interaction, Trade, TradeExecution};
 use crate::{encoding::EncodedSettlement, interactions::UnwrapWethInteraction};
-use anyhow::{anyhow, bail, ensure, Result};
+use anyhow::{bail, ensure, Context as _, Result};
```

Comment:
Similar question here, I can't see that `_` is actually used anywhere. What is the meaning of this?

Reply:
:point_up:

Reply:
So now that I know this, doesn't that mean there are probably a bunch of places in the code we could be doing this that clippy doesn't complain about?

Reply:
Yeah, but its not a big deal IMO (and doesn't change what the code actually does at all). Its just a habit.